src/OpenTelemetry.Contrib.Shared/OpenTelemetry.Contrib.Shared.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
```

Comment:
we probably don't need this or it should come from common props file.

Reply:
`Common.props` does not contain a `TargetFrameworks` element so this is required. Each instrumentation lib should probably choose it's own targets based on the library it's instrumenting.